src/expr/Cargo.toml

```diff
@@ -56,3 +56,5 @@ datadriven = "0.6.0"
 mz-expr-test-util = { path = "../expr-test-util" }
 mz-ore = { path = "../ore" }
 proc-macro2 = "1.0.36"
+proptest = { version = "1.0.0", default-features = false, features = ["std"] }
+proptest-derive = "0.2.0"
```

Comment:
`proptest-derive` is not yet used anywhere else AFAICT. Is there some compliance process new dependencies need to go through? This is a dev-dependency, if that makes a difference.

Reply:
Yes, the crate description states that the crate is only passively maintained, so I don't expect them to upgrade anytime soon. I briefly looked into what it would take to make a PR for them, but unfortunately there are significant breaking changes between the proc-macro versions in question and I don't know enough about proptest or proc macros to whip something up quickly.

Reply:
Ok, give me a few!

Reply:
I have a fork that compiles but is otherwise untested. Can you update all references to `proptest` and `proptest-derive` to point at `git = "https://github.com/materializeInc/proptest.git"` and see if it works?

Reply:
Everything looks good :)

Are you planning to make a PR to upstream? Then I can add that as context to the `allow-git` config.